scvid/data/distributed_anndata.py

```diff
@@ -307,12 +305,3 @@ def __repr__(self) -> str:
                 if len(keys) > 0:
                     descr += f"\n    {attr}: {str(list(keys))[1:-1]}"
         return descr
-
-
-@get_anndata_attribute.register
```

Comment:
For my education -- what happened here? why removed it? :)

Reply:
This is the function that is used by `AnnTorchDataset` in `scvi-tools`. In our fork I made it single-dispatch and registered its implementation for `AnnCollection`. Since we don't use `scvi-tools` training machinery we don't need this.